### PR TITLE
Configure Spring in Cinema project

### PR DESCRIPTION
**Pull request by Alkey**

### CODE REVIEW

File: src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-import com.dev.cinema.lib.Service;
 import com.dev.cinema.model.User;
 import com.dev.cinema.service.ShoppingCartService;
 import com.dev.cinema.service.UserService;
 import com.dev.cinema.util.HashUtil;
 import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 @Service
```
Comment (frostpv) [inline src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java:12]: @Component аннотация бы сюда лучше подошла

**Review by frostpv — APPROVED** [review]
Goog job

File: pom.xml [hunk 2]
```diff
@@ -43,53 +45,64 @@
...
+            <artifactId>jackson-databind</artifactId>
+            <version>2.10.2</version>
+        </dependency>
     </dependencies>
 
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-checkstyle-plugin</artifactId>
```
Comment (Sofasmile) [inline pom.xml]: why did you remove checkstyle plugin?

File: src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java (hunk 1 above)
Comment (Sofasmile) [reply src/main/java/com/dev/cinema/security/AuthenticationServiceImpl.java:12]: Service is ok too

File: pom.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <groupId>org.example</groupId>
     <artifactId>Cinema</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <packaging>war</packaging>
+<!--    <packaging>war</packaging>-->
```
Comment (Sofasmile) [inline pom.xml]: do we need this commented code?

**Review by Sofasmile — APPROVED** [review]
